test/reactTags.test.js

```diff
@@ -125,20 +125,21 @@ describe('Test ReactTags', () => {
     expect(actual).to.not.have.members(['Banana']);
   });
 
-  test('handles the paste event and splits the clipboard on delimiters', () => {
+  test('handles the paste event, splits the clipboard on delimiters, and ignores duplicates', () => {
```

Comment:
can you move the two tests related to paste event inside a new describe block 'tests handlePaste`